awscli/examples/schemas/update-discoverer.rst

```diff
@@ -0,0 +1,20 @@
+**To update an existing discoverer on the Event Bus**
+
+The following ``update-discoverer`` example updates an existing discoverer on the Event Bus. You can change the description, or you can enable or disable discovering schemas on cross-012345678912 Events. ::
+
+    aws schemas update-discoverer \
+        --discoverer-id example-discoverer-id \
+        --no-cross-012345678912
+
+Output::
+
+    {
+        "DiscovererArn": "arn:aws:schemas:us-east-1:012345678912:discoverer/example-discoverer-id",
+        "DiscovererId": "example-discoverer-id",
+        "SourceArn": "arn:aws:events:us-east-1:012345678912:event-bus/example-event-bus",
+        "State": "STOPPED",
+        "Cross012345678912": false,
```

Comment:
Looks like "account" may have been replaced with `012345678912` in three places.

Reply:
I'm still seeing `cross-012345678912` and `Cross012345678912` instead of `cross-account` and `CrossAccount`.